src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Tests/Fixtures/TimePeriodMetaGeneratorExtensions.cs

```diff
@@ -0,0 +1,41 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Fixtures;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Tests.Fixtures;
+
+public static class TimePeriodMetaGeneratorExtensions
+{
+    private const int DefaultStartYear = 2015;
```

Comment:
maybe start at `2000` like in the other place? I.e. `DataSetVersionMetaSummaryGeneratorExtensions`

Reply:
Yep, fair point!